### PR TITLE
Feature/print format update

### PR DESCRIPTION
**Pull request by Saralrat**
1. Edit form name from Thai to English
Screenshot from 2024-09-23 14-24-37

2. Create Print format of Quotation and Sale Order
http://redmine.ecosoft.co.th:10083/issues/5708

- Example Quotation 
Quotation MH

- Example Sales Order
Sales Order MH

3. Create Print format of Purchase Order
http://redmine.ecosoft.co.th:10083/issues/5712

- Example Purchase Order
Purchase Order MH

4. Create Print format of Purchase Receipt
http://redmine.ecosoft.co.th:10083/issues/5770

- Example Purchase Receipt
Purchase Receipt MH

5.Create Print format of Expense Claims 
http://redmine.ecosoft.co.th:10083/issues/5771

- Example Expense Claims 

Expense Claims MH

### CODE REVIEW

**Comment by newtratip** [comment]
I tried printing a quotation form where the content spans multiple pages and found that the text overlaps, as shown in the picture.
Selection_341

Please check this issue for all the forms you have created.

File: menghua_co/hooks.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 app_description = "Menghua Co\'s ERP"
 app_email = "kittiu@ecosoft.co.th"
 app_license = "mit"
+required_apps = ["erpnext", "thai_tax"]
```
[the remark this answers is not in the document]
Comment (kittiu) [reply menghua_co/hooks.py:7]: ต้องด้วยหรอ พี่เข้าใจว่าไม่จำเป็นนะ หรือว่ามันเป็นปัญหาของ v14 ?

**Review by kittiu — APPROVED** [review]

File: menghua_co/hooks.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (newtratip) [reply menghua_co/hooks.py:7]: ถ้าไม่ใส่จะเจอ Error นี้ครับ
image

**Comment by newtratip** [comment]
> @newtratip I think you can merge and deploy, and let BA test it. You will manage the deployment, right?

Yes, @Saralrat is fixing something on form, after she done i will deploy it.

File: menghua_co/hooks.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (kittiu) [reply menghua_co/hooks.py:7]: ป๊อดจะบอกว่า ถ้าใส่แบบป๊อด ถ้าไม่มี app อยู่แล้ว มันจะไป download มาให้เลย?

**Comment by Saralrat** [comment]
> > @newtratip I think you can merge and deploy, and let BA test it. You will manage the deployment, right?
> 
> Yes, @Saralrat is fixing something on form, after she done i will deploy it.
@newtratip I have fixed the header that was not in line with the form.

**Pull request merged**